Src/DependencyCollector/Net45.Tests/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
don't need this file

Reply:
I need it for binding redirects as tests and dependency collector reference different versions of DiagnosticsSource